### ISSUE TITLE
[Problem - BUG] Match crashed and not counted for

### ISSUE THREAD

**Issue by vasilegherasim** [IroncladGames/SinsOfADarkAge#963]
Had beed in a 5vs5, and it crashed, also the rejoin option wasn't there, and the performance was bad even if it was 100ms/60fps.
Log attached, just rename to txt.

![log-mar28_15_51_50](https://cloud.githubusercontent.com/assets/10153234/6881429/2f09572e-d568-11e4-8083-8b1ae4183a5f.jpg)
![untitled](https://cloud.githubusercontent.com/assets/10153234/6881430/2f0afe58-d568-11e4-829b-c2544727789e.jpg)

**Comment by uncannymanny** [comment]
Thanks for the report!

Hopefully the devs can look into what happened.

**Comment by vasilegherasim** [comment]
Happened 2 times again yesterday.

**Comment by jseward** [comment]
was 309663 the game that was bugged?

**Comment by jseward** [comment]
just to be clear the game WAS counted for? the bug description should be just game crashed, couldn't rejoin?

**Comment by vasilegherasim** [comment]
No, the game wasn't counted for, it doesn't appear on that list, but i notice that the other players had the same issue.

**Comment by uncannymanny** [comment]
Similar to http://steamcommunity.com/app/251970/discussions/2/611701999518175603/
